### PR TITLE
Fix rubocop warnings

### PR DESCRIPTION
**Pull request by josemigallas**
🚨

### CODE REVIEW

File: app/lib/system/deploy.rb [hunk 1]
```diff
@@ -71,8 +70,8 @@ def self.call(_)
 
     def self.load_info!(deploy_info = parse_deploy_info)
       self.info = Info.new(deploy_info)
-    rescue StandardError => error
-      self.info = InvalidInfo.new(error)
+    rescue StandardError => e
```
Comment (Martouta) [inline app/lib/system/deploy.rb:73]: why on earth would rubocop say that `e` is clearer than `error` ? 😕

**Review by Martouta — APPROVED** [review]
okidoki 😄

File: app/lib/system/deploy.rb (hunk 1 above)
Comment (josemigallas) [reply app/lib/system/deploy.rb:73]: It's not about clarity but naming convention. We could change it to `error` or disable it altogether.

File: Gemfile.lock [hunk 2]
```diff
@@ -630,13 +630,13 @@ GEM
       rspec (~> 3.0)
     rspec_junit_formatter (0.4.1)
       rspec-core (>= 2, < 4, != 2.12.0)
-    rubocop (0.93.1)
+    rubocop (0.92.0)
```
Comment (Martouta) [inline Gemfile.lock:633]: Why?
image

Comment (josemigallas) [reply Gemfile.lock:633]: So it's aligned with https://github.com/3scale/porta/pull/2265/files#diff-e97904d8a0443f46ad2c8ceb5fbe5355e52f6be540378ab8ea1e98870395d7deR15

Comment (josemigallas) [reply Gemfile.lock:633]: I'm trying to figure out why Circleci throws an error locally but passes here

Comment (guicassolato) [reply Gemfile.lock:626]: Can we try going v1.x? E.g.

https://rubygems.org/gems/rubocop/versions/1.9.1
https://github.com/codeclimate/codeclimate-rubocop/branches/all?query=channel%2Frubocop-1-9

**Comment by hallelujah** [comment]
Sorry, it seems your work did not get enough attention from us.
Thank you for making this project better.

We are going to close it as it was stale for too long.

**Pull request closed without merging**